### PR TITLE
Add io.github.tdesktop_x64.TDesktop

### PR DESCRIPTION
**Pull request by tehcneko**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/TDesktop-x64/tdesktop/commit/a0b1e82104c6cf91640bde58e07e32f9c8a4351f
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop-x64

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop-x64

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop-x64

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop-x64.desktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop-x64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop-x64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop64.TDesktop

File: io.github.tdesktop64.TDesktop.yml [hunk 1]
```diff
@@ -0,0 +1,554 @@
+id: io.github.tdesktop64.TDesktop
```
Comment (hfiguiere) [inline io.github.tdesktop64.TDesktop.yml:1]:
```suggestion
id: io.github.tdesktop_x64.TDesktop
```

This would be the correct app-id

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Comment by hfiguiere** [comment]
why do you need to build Qt?

File: qtbase.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "type": "patch",
+    "paths": [
+        "desktop-app-patches/qtbase_6.5.2/0001-spellcheck-underline-from-chrome.patch",
+        "desktop-app-patches/qtbase_6.5.2/0002-improve-apostrophe-processing.patch",
+        "desktop-app-patches/qtbase_6.5.2/0003-fix-shortcuts-on-macos.patch",
```
Comment (hfiguiere) [inline qtbase.json:6]: irrelevant

File: qtbase.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        "desktop-app-patches/qtbase_6.5.2/0004-allow-creating-floating-panels-macos.patch",
+        "desktop-app-patches/qtbase_6.5.2/0005-fix-file-dialog-on-windows.patch",
```
Comment (hfiguiere) [inline qtbase.json:8]: irrelevant

File: qtbase.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        "desktop-app-patches/qtbase_6.5.2/0006-fix-launching-mail-program-on-windows.patch",
```
Comment (hfiguiere) [inline qtbase.json:9]: irrelevant

File: qtbase.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        "desktop-app-patches/qtbase_6.5.2/0007-fix-gtk-file-dialog-ubuntu-1404.patch",
```
Comment (hfiguiere) [inline qtbase.json:10]: pretty sure it's irrelevant

File: qtbase.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        "desktop-app-patches/qtbase_6.5.2/0008-save-dirtyopaquechildren.patch",
+        "desktop-app-patches/qtbase_6.5.2/0009-old-freetype-compatibility.patch",
```
Comment (hfiguiere) [inline qtbase.json:12]: freetype is not old on the flatpak runtime

File: qtbase.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+        "desktop-app-patches/qtbase_6.5.2/0010-always-use-xft-font-conf.patch",
+        "desktop-app-patches/qtbase_6.5.2/0011-catch-cocoa-dock-menu.patch",
+        "desktop-app-patches/qtbase_6.5.2/0012-fix-race-in-windows-timers.patch",
+        "desktop-app-patches/qtbase_6.5.2/0013-nicer-platformtheme-choosing.patch",
+        "desktop-app-patches/qtbase_6.5.2/0014-reset-current-context-on-error.patch",
+        "desktop-app-patches/qtbase_6.5.2/0015-reset-opengl-widget-on-context-loss.patch",
+        "desktop-app-patches/qtbase_6.5.2/0016-qtgui-dlopen.patch",
+        "desktop-app-patches/qtbase_6.5.2/0017-fix-imes-on-macos.patch",
+        "desktop-app-patches/qtbase_6.5.2/0018-fix-search-in-finder-macos.patch",
```
Comment (hfiguiere) [inline qtbase.json:21]: irrelevant (lines 20-21)

File: qtbase.json [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+        "desktop-app-patches/qtbase_6.5.2/0003-fix-shortcuts-on-macos.patch",
+        "desktop-app-patches/qtbase_6.5.2/0004-allow-creating-floating-panels-macos.patch",
+        "desktop-app-patches/qtbase_6.5.2/0005-fix-file-dialog-on-windows.patch",
+        "desktop-app-patches/qtbase_6.5.2/0006-fix-launching-mail-program-on-windows.patch",
+        "desktop-app-patches/qtbase_6.5.2/0007-fix-gtk-file-dialog-ubuntu-1404.patch",
+        "desktop-app-patches/qtbase_6.5.2/0008-save-dirtyopaquechildren.patch",
+        "desktop-app-patches/qtbase_6.5.2/0009-old-freetype-compatibility.patch",
+        "desktop-app-patches/qtbase_6.5.2/0010-always-use-xft-font-conf.patch",
+        "desktop-app-patches/qtbase_6.5.2/0011-catch-cocoa-dock-menu.patch",
+        "desktop-app-patches/qtbase_6.5.2/0012-fix-race-in-windows-timers.patch",
```
Comment (hfiguiere) [inline qtbase.json:15]: irrelevant (lines 14-15)

**Comment by hfiguiere** [comment]
why does this fork Qt?

**Comment by tehcneko** [comment]
> why does this fork Qt?

This is a forked project(64Gram), not forking Qt

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Comment by tehcneko** [comment]
@hfiguiere Is there anything more that requires changes?

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Comment by tehcneko** [comment]
bot, build io.github.tdesktop_x64.TDesktop

**Pull request closed without merging**